packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1416,6 +1427,13 @@ export class ContainerRuntime
 			summaryFormatVersion: metadata?.summaryFormatVersion,
 			disableIsolatedChannels: metadata?.disableIsolatedChannels,
 			gcVersion: metadata?.gcFeature,
+			options: JSON.stringify(runtimeOptions),
+			featureGates: JSON.stringify({
```

Comment:
I wonder, is there a way to enumerate all the gates out there, even if code does not know about their names? Seems like better direction (maybe we attack it as separate task)

Reply:
Yes! I mean there should be, and it has been an ask about this, which I explored briefly but ran into some chicken-and-egg problems about logging stuff within the logger (mixin config logger, actually). The gist of it is that we can capture all feature gate 'get' calls and log the first time they're accessed.

Unfortunately, there is no central place for feature gates to be declared, but there probably should be, something similar to what we do with the assert codes.

Reply:
Opened ADO:3466

Reply:
We should double check with Loop (the bottom of the stack). I hope there is a way to enumerate all feature gates, as they are known (i.e. we get whole package from ECS, it's not call-per-call retrieval).
Logging get calls will not work - some of the get calls may happen minutes into the session.